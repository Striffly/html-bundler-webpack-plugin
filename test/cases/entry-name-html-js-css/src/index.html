<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="./index.css" rel="stylesheet">
  <script src="./index.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>