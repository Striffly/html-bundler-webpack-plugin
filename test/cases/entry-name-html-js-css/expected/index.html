<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="index.bundle.css" rel="stylesheet">
  <script src="index.bundle.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>